Turf.xcodeproj/project.pbxproj

```diff
@@ -27,6 +38,9 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		353E9B011F3E08BD007CFA23 /* Turf copy-Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = "Turf copy-Info.plist"; path = "/Users/fredrik/Workspace/mapbox/turf-swift/Turf copy-Info.plist"; sourceTree = "<absolute>"; };
```

Comment:
I overlooked this. Creating a reference or copying while dragging is randomly picked in Xcode 9 beta 1-6. The recommended workaround is to hold down <kbd>⌘</kbd><kbd>⌥</kbd> while dragging for just creating a ref. Anyways, good catch.